tool/tctl/common/resource_command.go

```diff
@@ -419,6 +438,60 @@ func (rc *ResourceCommand) Delete(client auth.ClientI) (err error) {
 	return nil
 }
 
+// Update updates select resource fields: expiry and labels
+func (rc *ResourceCommand) Update(clt auth.ClientI) error {
+	if rc.ref.Kind == "" || rc.ref.Name == "" {
+		return trace.BadParameter("provide a full resource name to update, for example:\n$ tctl update rc/remote --set-labels\n")
+	}
+
+	var err error
+	var labels map[string]string
+	if rc.labels != "" {
+		labels, err = client.ParseLabelSpec(rc.labels)
+		if err != nil {
+			return err
+		}
+	}
+
+	var expiry *time.Time
```

Comment:
you can use a non-pointer `time.Time` and call `expiry.IsZero()` to see if it's set

Reply:
ping